hawc/apps/lit/constants.py

```diff
@@ -32,3 +33,7 @@ class SearchType(models.TextChoices):
 DOI_EXACT = re.compile(r"^10\.\d{4,9}/[^\s]+$")
 DOI_EXTRACT = re.compile(r"10\.\d{4,9}/[^\s]+")
 DOI_EXAMPLE = "10.1234/s123456"
+
+REFERENCE_SEARCH_VECTOR = SearchVector(
+    "abstract", "title", "authors", "year", "journal", config="english"
```

Comment:
its redundant, but I think we should add `authors_short` too; sometimes people put other stuff in that one